edxval/api.py

```diff
@@ -1,69 +1,31 @@
 # pylint: disable=E1101
 # -*- coding: utf-8 -*-
 """
-The internal API for VAL. This is not yet stable
```

Comment:
is there any context behind `This is not yet stable` which has been removed from L4